supreme_court_predictions/api/convokit/client.py

```diff
@@ -0,0 +1,8 @@
+"""
+This file serves as the client for convokit.
+"""
+
+
+def get_data():
```

Comment:
@chanteriam, here's the file to add getting the data.

Reply:
Ideally, we'd download it and then unzip it in the `data/convokit` folder.

Reply:
With this function, do I need to keep the main() function? @michplunkett

Reply:
You shouldn't need it, no.